content/en-us/reference/engine/enums/SurfaceType.yaml

```diff
@@ -31,70 +28,68 @@ description: |
   - `Enum.JointCreationMode`, the way joints are created between two surfaces
 code_samples:
 tags: []
-deprecation_message: ''
+deprecation_message: |
+  SurfaceType based joining is deprecated, leaving only visual changes to the effected 'Class.Part'. This should not be used for future work.
 items:
   - name: Smooth
     summary: |
-      Makes the side appear without any surface detail (except for _outlines_).
+      Adds no details on the surface.
     value: 0
     tags: []
     deprecation_message: ''
   - name: Glue
     summary: |
-      Makes the side appear with thick diagonal "X"s.
+      Adds an X pattern across the surface.
     value: 1
     tags: []
     deprecation_message: ''
   - name: Weld
     summary: |
-      Makes the side appear with thick diagonal "X"s.
+      Adds an X pattern across the surface.
     value: 2
     tags: []
     deprecation_message: ''
   - name: Studs
     summary: |
-      Makes the side appear with square studs.
+      Adds square studs across the surface.
     value: 3
     tags: []
     deprecation_message: ''
   - name: Inlet
     summary: |
-      Makes the side appear with holes where studs would be.
+      Adds square holes across the surface where studs would be.
     value: 4
     tags: []
     deprecation_message: ''
   - name: Universal
     summary: |
-      Makes the side appear with both Studs and Inlets in a checker pattern.
+      Adds a checker pattern to the surface using Studs and Intlets.
     value: 5
     tags: []
     deprecation_message: ''
   - name: Hinge
     summary: |
-      Makes the side appear with a yellow hinge. Any part connected to this
-      hinge will stick to the side and rotate using physics, however, using
-      `Class.HingeConstraint` to join parts is preferred.
+      Adds a yellow hinge to the surface. Parts touching it would stick to the surface allowing for rotations using physics. This should not be used for future work, and should be replaced with a `Class.HingeConstraint` instead.
```

Comment:
```suggestion
      Added a yellow hinge to the surface. Parts touching it would stick to the surface allowing for rotations using physics. This should not be used for future work and existing instances should be replaced with a `Class.HingeConstraint`.
```